src/app/components/user/hidUsers.controller.js

```diff
@@ -0,0 +1,135 @@
+hidControllers.controller('hidUsersCtrl', ['$scope', '$rootScope', '$location', '$routeParams', 'gettextCatalog', 'alertService', 'hrinfoService', 'userService', 'User', 'List',
```

Comment:
Ok good point, I don't know why I chose this name... Let's call it UsersCtrl and rename the existing UsersCtrl into UsersPageCtrl if that's ok with you.